FLEDGE.md

```diff
@@ -643,7 +643,8 @@ Reports are only sent and most interest group state changes (e.g. updating `prev
 
 Once the winning ad has rendered in its Fenced Frame, the seller and the winning buyer each have an opportunity to perform logging and reporting on the auction outcome.  The browser will call one reporting function in the seller's auction worklet and one in the winning buyer's bidding worklet.
 
-_As a temporary mechanism,_ these reporting functions will be able to send event-level reports to their servers.  These reports can include contextual information, and can include information about the winning interest group if it is over an anonymity threshold.  This reporting will happen synchronously, while the page with the ad is still open in the browser.
+_As a temporary mechanism,_ these reporting functions will be able to send event-level reports to their servers.  These reports can include contextual information, and can include information about the winning interest group if it is over an anonymity threshold.
+The worklet functions `reportWin()` and `reportResult()` may call a `sendReportTo()` API which takes a single string argument representing a URL. The `sendReportTo()` function should be called exactly once during a worklet function's execution. The URL will is fetched after the reporting worklet completes.  This fetch will happen synchronously, while the page with the ad is still open in the browser.
 
 In the long term, we need a mechanism to ensure that the after-the-fact reporting cannot be used to learn the advertising interest group of individual visitors to the publisher's site — the same privacy goal that led to Fenced Frame rendering. The [Private Aggregation API](https://github.com/alexmturner/private-aggregation-api) proposal aims to satisfy this use case. Therefore event-level reporting is just a temporary model until an adequate reporting framework is settled and in place.
 
```

Comment:
We need to change the two references below to "network APIs" to instead reference `sendReportTo()`.  I'm wondering if we should leave the "5." section as high-level prose not getting into `sendReportTo()` specifics, and move the explanation of `sendReportTo()` down to the first "network APIs" references in "5.1", and then make "5.2" reference the explanation of `sendReportTo()` in section "5.1".  I don't feel strongly about where we put the explanation of `sendReportTo()`, but we should remove the reference to "network APIs".